test/component/test_component.py

```diff
@@ -28,12 +29,10 @@ def test_e2e_with_sucessful_msg(self):
         expected_tx_id = str(uuid.uuid4())
         self.publish_to_pubsub(expected_tx_id, expected_case_id)
 
-        expected_msg = (
-            f'<?xml version="1.0" encoding="UTF-8" standalone="yes"?>'
-            f'<ns2:caseReceipt xmlns:ns2="http://ons.gov.uk/ctp/response/casesvc/message/feedback">'
-            f'<caseId>{expected_case_id}</caseId><inboundChannel>OFFLINE</inboundChannel>'
-            f'<responseDateTime>2008-08-24T00:00:00+00:00</responseDateTime></ns2:caseReceipt>'
-        )
+        expected_msg = json.dumps({'case_id': expected_case_id,
```

Comment:
For the benefit of your future selves might it be worth adding questionnaire_id in here as well? Then it's clear where that value is expected to appear.